### PR TITLE
Parse uk

### PR DESCRIPTION
**Pull request by NeonMariia**

### CODE REVIEW

**Comment by JarbasAl** [comment]
we just had a stable release and are back in alpha versions, I am ok with merging this as is and fix any issues in follow up PRs. This is great work and I'd like to get it in the wild!

Only see some debug prints to remove and maybe a rebase, there seems to be some duplicated code too between this and #50

File: lingua_franca/internal.py [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
 _SUPPORTED_LANGUAGES = ("az", "ca", "cs", "da", "de", "en", "es", "fr", "hu",
-                        "it", "nl", "pl", "pt", "ru", "sl", "sv", "fa", "eu")
+                        "it", "nl", "pl", "pt", "ru", "sl", "sv", "fa", "eu", "uk")
 
 _SUPPORTED_FULL_LOCALIZATIONS = ("az-az", "ca-es", "cs-cz", "da-dk", "de-de",
                                  "en-au", "en-us", "es-es", "fr-fr",
                                  "hu-hu", "it-it", "nl-nl", "pl-pl",
                                  "fa-ir", "pt-pt", "ru-ru", "sl-si",
-                                 "sv-se", "tr-tr", "eu-eu")
+                                 "sv-se", "tr-tr", "eu-eu", "uk-uk")
```
Comment (NeonBohdan) [inline lingua_franca/internal.py:19]: Lang code is uk-ua
See [here](http://www.lingoes.net/en/translator/langcode.htm)

**Comment by NeonDaniel** [comment]
#56 can replace this PR, or I can force push those changes to @NeonMariia 's branch attached to this PR.

The commit history for parse and format had some shared commits, so I cherry-picked only the new ones for #56; tests are passing, so I think I managed the conflicts correctly

**Comment by NeonDaniel** [comment]
Replaced by #56

**Pull request closed without merging**